src/pages/docs/worker-bridge.mdx

```diff
@@ -0,0 +1,452 @@
+## Worker Bridge
+
+Worker-Bridge, an integral component of the `Golem OSS` worker-service, empowers you to define and upload API definitions.
+These definitions allow you to expose endpoints to users and  specify how incoming requests should be managed by your `Golem` worker workflow/application.
+
+Let's dig a bit deeper using an example.
+
+Say you already have a worker running in your `Golem OSS` (docker environment) or `Golem Cloud`.  Assuming you have already got the worker running in `Golem` with shopping-cart example,
+let's say we have to expose an endpoint to get the cart contents for a user.  In order to do this, you would typically write an API definition as given below and simply upload to `Golem`.
+
+
+At this point, worker-bridge is acting as an API-Gateway, but the major difference here is, it is geared
+to work seamlessly with `Golem`. In other words, we don't need to specifically write a custom backend service as worker-bridge gives it for free.
+
+Here is a simple API definition. You have to replace with the correct template-id and function name once you update your shopping-cart template to golem. Consider the above
+as a simple example.
+
+```json
+
+{
```

Comment:
Or is this compliant OpenAPI??

Reply:
I removed the code that handled a custom yaml file.
This custom json to be user facing can also be removed, but note that the purpose of supporting OpenAPI spec is explained further below. 

Forcing users to write OpenAPI spec just to feed the below information is also not ideal. The support of open-api is done so that, if users already have an OpenAPI spec describing their endpoints, they can edit the same spec to feed in the worker-bridge info using extensions and upload the same to worker-bridge